services/fuel-core/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/fuellabs/fuel-core:v0.4.1
+FROM ghcr.io/fuellabs/fuel-core
```

Comment:
Shouldn't this be kept pinned? Otherwise a new breaking release of `fuel-core` will break all CI flows here.